Maya2glTF.vcxproj

```diff
@@ -40,6 +48,13 @@
     <CharacterSet>MultiByte</CharacterSet>
     <MayaVersion>2018</MayaVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Maya2019-Debug|x64'" Label="Configuration">
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <PlatformToolset>v141</PlatformToolset>
+    <CharacterSet>MultiByte</CharacterSet>
+    <MayaVersion>2018</MayaVersion>

```

Comment:
I just noticed this. I think these might still need to be refactored to 2019 even though compilation worked on my end. It's the same in the `Maya2019-Release|x64` property group.